src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XNode.cs

```diff
@@ -138,7 +138,7 @@ public void AddAfterSelf(object? content)
         /// <exception cref="InvalidOperationException">
         /// Thrown if the parent is null.
         /// </exception>
-        public void AddAfterSelf(params object[] content)
+        public void AddAfterSelf(params object?[] content)
```

Comment:
I think this triggers subtle nullability changes on the following snippet which I don't see anywhere in this PR.
https://github.com/dotnet/runtime/blob/1cc8b1c83ff6508dd8b428567e2160a8fd5a5081/src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XLinq.cs#L98-L109